inc/templates.php

```diff
@@ -478,18 +480,14 @@ function handle_data_provider( $component ) {
  * @param array $component Component.
  * @return array
  */
-function handle_component_config_callbacks( $component ): array {
+function handle_component_config_callbacks( $component ): Component {
```

Comment:
We should be able to deprecate this in favor of the coming context API.

Reply:
How so? I don't see how the context api removes the need for a render callback?

Reply:
Maybe I'm misunderstanding the use of this function. I thought the responsibility of this function was to convert the mustache style config values to computed values.

Reply:
Right, I'm getting my wires crossed. And yes, we'll be deprecating soon. Going to leave as-is for now though.